x-pack/plugins/security_solution/server/lib/detection_engine/signals/search_after_bulk_create.ts

```diff
@@ -64,16 +62,6 @@ export const searchAfterAndBulkCreate = async ({
   // to ensure we don't exceed maxSignals
   let signalsCreatedCount = 0;
 
-  const totalToFromTuples = getSignalTimeTuples({
```

Comment:
Rather than calculate the tuples inside `searchAfterBulkCreate`, they are calculated in `signal_rule_alert_type.ts` and passed into `searchAfterBulkCreate`.